jujugui/static/gui/src/test/test_browser_charm_details.js

```diff
@@ -330,9 +330,11 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
     it('_addCharmEnvironment displays the config panel', function(done) {
-      var fakeStore = new Y.juju.charmstore.APIv4({
-        charmstoreURL: 'localhost/'
-      });
+      window.juju_config = {
```

Comment:
If you're going to modify something in the global state be sure you set it back to what it was before you modified it in an afterEach so that you know that no matter what it'll get reset even if the test fails. ( repeat for all instances below )